src/test/scala/nl.knaw.dans.easy.dd2d/queue/ActiveTaskQueueSpec.scala

```diff
@@ -15,13 +15,16 @@
  */
 package nl.knaw.dans.easy.dd2d.queue
 
+import nl.knaw.dans.easy.dd2d.Deposit
+import nl.knaw.dans.lib.taskqueue.ActiveTaskQueue
 import org.scalatest.{ FlatSpec, Matchers, OneInstancePerTest }
 
 class ActiveTaskQueueSpec extends FlatSpec with Matchers with OneInstancePerTest {
 
   // TODO: how to make this test robust?
+  //TODO: should this be moved to dans-scala-lib?
```

Comment:
Answer: definitely! Please remove it from the project. I will add it to dans-scala-lib.